easybuild/easyconfigs/t/TOBIAS/TOBIAS-0.16.1-foss-2023a.eb

```diff
@@ -46,8 +46,6 @@ exts_list = [
         'checksums': ['a74408f69ba6271f71b9352ef4ed03dc53a31aa404d29b5d31f53bfecfee1440'],
     }),
     ('tobias', version, {
-        # remove pyBigWig dependency - pip_check fails with "import pybigwig"
-        'preinstallopts': "sed -i '81d' setup.py && ",
```

Comment:
Just to clarify: this can be removed because by using the pyBigWig source tarball from PyPI rather than GitHub, the necessary metadata is there so `pip` is aware of the `pyBigWig` installation (`pip check` doesn't complain, `pip list` shows it)